ignite/cmd/scaffold_module.go

```diff
@@ -17,9 +17,7 @@ import (
 )
 
 // moduleNameKeeperAlias is a map of well known module names that have a different keeper name than the usual <module-name>Keeper.
-var moduleNameKeeperAlias = map[string]string{
-	"auth": "account",
```

Comment:
This won't make sense too long anymore with `x/accounts` coming soon and v0.50 compatible